docker-compose.yml

```diff
@@ -28,7 +28,7 @@ services:
       - postgres
       - redis
     ports:
-      - "5000:5000"
+      - "5001:5000"
```

Comment:
Probably want to update this in https://github.com/getredash/redash/blob/fdf636a3935b0d42e87c5a3e0b543f748f871392/webpack.config.js#L37 too. And in the local Cypress config (but this one I need to confirm still)

Reply:
Nice catch. I'll grep the code for `5000`.